Signal.xcodeproj/project.pbxproj

```diff
@@ -2449,7 +2453,7 @@
 					"\"$(SRCROOT)/Libraries\"/**",
 				);
 				INFOPLIST_FILE = "$(SRCROOT)/Signal/Signal-Info.plist";
-				IPHONEOS_DEPLOYMENT_TARGET = 8.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 9.0;
```

Comment:
Huh, I forgot that I included some of this in: https://github.com/WhisperSystems/Signal-iOS/commit/42768294eebda42c1d2f060e0f5ff70770645d0b